### PR TITLE
Added in new feature Skins (stylesheets) 

### PR DESCRIPTION
**Pull request by laurenyew**
Skins are stylesheets.
Added in database for Skin (Skin.java) and controller for Skins (Skins.java) as well as basic mapping of Skins to user.
Currently, all pages get the rendered skin attached to the current connected user.
Unit Testing: test/unit/SkinTest.java

### CODE REVIEW

**Comment by amshali** [comment]
As we discussed, please send another commit.
On Mar 1, 2012 11:03 PM, "Lauren Yew" <
reply@reply.github.com>
wrote:

File: .project [hunk 1]
```diff
@@ -15,11 +15,12 @@
```
Comment (amshali) [inline .project:-1]: Could you please remove .project from your commit?
Add the followings to your .gitignore file

/tmp
/modules
/lib
/test-result
/logs
/eclipse
.classpath
/.settings
*.swp
*~
*.log
data/
.project

Comment (laurenyew) [inline .project:-1]: I added to the .gitignore file. My git is now saying that I am ignoring the project, but it's not getting rid of this when I redo the pull request. Not sure what that means.

**Comment by amshali** [comment]
How can I change the skin?

**Comment by laurenyew** [comment]
To change a skin, call Skins.setSkin(User u, String skinName).

**Comment by amshali** [comment]
Can you provide a UI for that?

-- Amin

On Sat, Mar 3, 2012 at 3:34 PM, Lauren Yew <
reply@reply.github.com

**Comment by laurenyew** [comment]
That is the feature that I am working on now. I am planning to add it to the edit profile page as a link to either a choose style sheet page which can set the style sheet to an already created one, or edit my style sheet, which will edit a new style sheet for that user.

File: .project (hunk 1 above)
Comment (ProCynic) [inline .project:-1]: adding something to .gitignore means that it won't be added to the repo.  It doesn't remove it if it's already present and it doesn't affect pulls.

**Pull request merged**